src/table/utils/copy-utils.ts

```diff
@@ -0,0 +1,14 @@
+const copyCellValue = async (evt: any, isHeadCell: boolean = false) => {
```

Comment:
I just realized we could follow the same pattern as other nebula hooks, so you can put this function in the use-context-menu file

Reply:
ok I'll move it